src/test/java/api/InstancesApiExamples.java

```diff
@@ -36,39 +38,21 @@
 import java.util.concurrent.ExecutionException;
 import java.util.concurrent.TimeoutException;
 
-import static api.support.InstanceSamples.leviathanWakes;
-import static api.support.InstanceSamples.marcInstanceWithDefaultBlockedFields;
-import static api.support.InstanceSamples.nod;
-import static api.support.InstanceSamples.smallAngryPlanet;
-import static api.support.InstanceSamples.taoOfPooh;
-import static api.support.InstanceSamples.temeraire;
-import static api.support.InstanceSamples.treasureIslandInstance;
-import static api.support.InstanceSamples.uprooted;
+import static api.support.InstanceSamples.*;
 import static io.vertx.core.http.HttpMethod.POST;
 import static io.vertx.core.http.HttpMethod.PUT;
 import static java.util.Arrays.asList;
 import static java.util.concurrent.TimeUnit.SECONDS;
-import static org.folio.inventory.domain.instances.Instance.PRECEDING_TITLES_KEY;
-import static org.folio.inventory.domain.instances.Instance.PUBLICATION_PERIOD_KEY;
-import static org.folio.inventory.domain.instances.Instance.TAGS_KEY;
-import static org.folio.inventory.domain.instances.Instance.TAG_LIST_KEY;
+import static org.folio.inventory.domain.instances.Instance.*;
 import static org.folio.inventory.domain.instances.PublicationPeriod.publicationPeriodToJson;
-import static org.hamcrest.CoreMatchers.hasItem;
-import static org.hamcrest.CoreMatchers.hasItems;
-import static org.hamcrest.CoreMatchers.is;
-import static org.hamcrest.CoreMatchers.notNullValue;
-import static org.hamcrest.CoreMatchers.nullValue;
-import static org.hamcrest.Matchers.contains;
+import static org.hamcrest.CoreMatchers.*;
```

Comment:
Whist FOLIO has no official standards on imports, in order to reduce the amount of unnecessary flux when different folks order them, some of us tend to use Eclipse import ordering, which does not allow wildcard import statements ([FOLIO-1667|https://issues.folio.org/browse/FOLIO-1667] has more information)

Reply:
@marcjohnson-kint  Thank you for this comment. I miss it in the setting of IDE after moving from another device. Removed wildcards.